### PR TITLE
Line break before declaration initial value

### PR DESCRIPTION
**Pull request by jablko**
### Before

```Solidity
string veryVeryVeryVeryVeryLongVariableCalledA = "The quick brown fox jumps over the lazy dog";
```

### After

```Solidity
string veryVeryVeryVeryVeryLongVariableCalledA =
    "The quick brown fox jumps over the lazy dog";
```

### CODE REVIEW

**Comment by mattiaerre** [comment]
same comment as https://github.com/prettier-solidity/prettier-plugin-solidity/pull/190#issuecomment-519379159

**Comment by jablko** [comment]
### Before

```Solidity
string veryVeryVeryVeryVeryLongVariableCalledA = "The quick brown fox jumps over the lazy dog";
```

### After

```Solidity
string veryVeryVeryVeryVeryLongVariableCalledA =
    "The quick brown fox jumps over the lazy dog";
```

File: tests/Arrays/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -10,20 +10,21 @@ contract Arrays {
...
-        bytes32("removeUserFromGroup"),
-        bytes32("addSolution"),
-        bytes32("removeSolution"),
-        bytes32("addLibrary"),
-        bytes32("removeLibrary"),
-        bytes32("addPermissionToGroup"),
-        bytes32("removePermissionFromGroup")
-    ];
+    bytes32[] public permissions =
+        [
```
Comment (fvictorio) [inline tests/Arrays/__snapshots__/jsfmt.spec.js.snap]: Are we sure we want this? I kind of prefer the previous version.

File: tests/SplittableCommodity/__snapshots__/jsfmt.spec.js.snap [hunk 2]
```diff
@@ -80,8 +80,8 @@ contract SplittableCommodity is MintableCommodity {
         public
         whenNotPaused
     {
-        address supplierProxy = IContractRegistry(contractRegistry)
-            .getLatestProxyAddr("Supplier");
+        address supplierProxy =
+            IContractRegistry(contractRegistry).getLatestProxyAddr("Supplier");
```
Comment (fvictorio) [inline tests/SplittableCommodity/__snapshots__/jsfmt.spec.js.snap]: Same comment as before.

**Comment by mattiaerre** [comment]
can you resolve the conflicts here too @jablko ? thanks

File: tests/Arrays/__snapshots__/jsfmt.spec.js.snap (hunk 1 above)
Comment (mattiaerre) [reply tests/Arrays/__snapshots__/jsfmt.spec.js.snap]: I agree w/ that

File: tests/Conditional/__snapshots__/jsfmt.spec.js.snap [hunk 3]
```diff
@@ -23,9 +23,8 @@ pragma solidity ^0.4.24;
 
 contract Conditional {
     function foo() {
-        address contextAddress = longAddress_ == address(0)
-            ? msg.sender
-            : currentContextAddress_;
+        address contextAddress =
+            longAddress_ == address(0) ? msg.sender : currentContextAddress_;
```
Comment (mattiaerre) [inline tests/Conditional/__snapshots__/jsfmt.spec.js.snap]: I like the previous version more

File: tests/FunctionCalls/__snapshots__/jsfmt.spec.js.snap [hunk 4]
```diff
@@ -23,8 +23,8 @@ contract FunctionCalls {
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 contract FunctionCalls {
     function foo() {
-        address veryLongValidatorAddress = veryVeryVeryLongSignature
-            .popLast20Bytes();
+        address veryLongValidatorAddress =
+            veryVeryVeryLongSignature.popLast20Bytes();
```
Comment (mattiaerre) [inline tests/FunctionCalls/__snapshots__/jsfmt.spec.js.snap]: this one is nicer for me in this case

**Comment by mattiaerre** [comment]
what do we wanna do w/ this PR @Janther @fvictorio @jablko ?

**Comment by fvictorio** [comment]
With these many conflicts, maybe it's easier to just do a new PR for this.

**Comment by Janther** [comment]
Personally I don't think that breaking splitting after the `=` sign is something we want.

**Comment by fvictorio** [comment]
> Personally I don't think that breaking splitting after the = sign is something we want.

FWIW prettier-js does it, if I remember correctly. Like with many things prettier does, I hated it at the beginning, but eventually got used to it.

**Comment by Janther** [comment]
> FWIW prettier-js does it, if I remember correctly. Like with many things prettier does, I hated it at the beginning, but eventually got used to it.

The way prettier does it is very specific. It only indents the values if the expression in the right is a binary operation. This one will break if it's a long function call as well.

**Comment by mattiaerre** [comment]
I'd be inclined to close this PR; what do you think @fvictorio @Janther ?

**Comment by fvictorio** [comment]
Yes, it's been a while and it has several merge conflicts. At this point, assuming this is something we want, it's just easier to make a new PR.

**Pull request closed without merging**